src/main/java/software/amazon/awssdk/crt/http/HttpClientConnectionManagerOptions.java

```diff
@@ -157,5 +158,18 @@ public HttpClientConnectionManagerOptions withProxyOptions(HttpProxyOptions prox
      * Gets the proxy options for connections in the connection pool
      */
     public HttpProxyOptions getProxyOptions() { return proxyOptions; }
+
+    /**
+     * If true, the read back pressure mechanism is enabled for connections.
+     */
+    public boolean isReadBackPressureEnabled() { return enableReadBackPressure; }
+
+    /**
+     * If true, the read back pressure mechanism is enabled for connections. Defaults to false
+     */
+    public HttpClientConnectionManagerOptions withReadBackPressureEnabled(boolean enableReadBackPressure) {
```

Comment:
Did you see my comment about this on crt-cpp?
https://github.com/awslabs/aws-crt-cpp/pull/104#discussion_r394011329